.github/workflows/test.yml

```diff
@@ -14,10 +14,34 @@ jobs:
 
     steps:
     - uses: actions/checkout@main
-    - uses: './'
+
+    - name: Build Lua
+      uses: './'
       with:
         luaVersion: ${{ matrix.luaVersion }}
+        ignoreCache: true
```

Comment:
instea of a 'negative' boolean here can we have it called `buildCache`, set to true by default to have it do the cache

Reply:
Done